README.md

```diff
@@ -98,6 +98,18 @@ Truncate the core to a smaller length.
 Per default this will update the fork id of the core to `+ 1`, but you can set the fork id you prefer with the option.
 Note that the fork id should be monotonely incrementing.
 
+#### `const stream = core.createReadStream([options])`
+
+Make a read stream. Options include:
+
+``` js
+{
+  start: 0,
+  end: core.length,
+  live: false
+}
```

Comment:
Is `opts.snapshot` public API that should be documented?

Reply:
Ya, I'm just not ready to explain it at this stage 🤣

Reply:
Fair enough 😂

Reply:
I documented it now, it was a bad excuse :)